### PR TITLE
[site] Fix small typos in docs and site info

### PR DESCRIPTION
**Pull request by dmvolod**
Signed-off-by: Dmitry Volodin <dmvolod@gmail.com>

## Description
Small fixes in the docs for typos and changing name of the `Getting started` section for russian language

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Fix small typos in docs and site info.
impact_level: low
```

### CODE REVIEW

File: modules/300-prometheus/crds/doc-ru-prometheusremotewrite.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ spec:
                         description: |
                           Лейбл, в который записывается результирующее значение при замене.
 
-                          Поле является обезательным при `action: replace`.
```
Comment (apolovov) [inline modules/300-prometheus/crds/doc-ru-prometheusremotewrite.yaml:60]: 🤦

File: docs/site/pages/deckhouse_vs_kaas_ru.html [hunk 2]
```diff
@@ -889,7 +889,7 @@ <h2 class="features__item-title text_h1">
           количество резервных увеличивается с 1 до 5.
         </li>
         <li>
-          Резервные узлы повышают устойчивость кластера при резком росте нагрузки: ВМ уже работают и, при необходимости,
+          Резервные узлы повышают устойчивость кластера при резком увеличении нагрузки: ВМ уже работают и, при необходимости,
```
Comment (apolovov) [inline docs/site/pages/deckhouse_vs_kaas_ru.html:892]: It is debatable, i like "росте".

Comment (dmvolod) [reply docs/site/pages/deckhouse_vs_kaas_ru.html:892]: Ok, agree. Thanks.
Looks like in some cases, `рост нагрузки` is common expression, but sometimes `рост нагрузки` is also utilizing
Will revert this change.

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3553](https://github.com/deckhouse/deckhouse/pull/3553) to the branch [release-1.42](https://github.com/deckhouse/deckhouse/tree/release-1.42) successful!

**Pull request merged**